tanstack-table/src/main/scala/lucuma/react/table/Cell.scala

```diff
@@ -5,12 +5,12 @@ package lucuma.react.table
 
 import lucuma.typed.tanstackTableCore as raw
 
-case class Cell[T, A](private val toJs: raw.buildLibTypesMod.Cell[T, A]):
-  lazy val id: CellId                                         = CellId(toJs.id)
-  lazy val row: Row[T]                                        = Row(toJs.row)
-  lazy val column: Column[T, A]                               = Column(toJs.column)
-  def getContext(): raw.buildLibCoreCellMod.CellContext[T, A] = toJs.getContext()
-  def getValue[V](): V                                        = toJs.getValue().asInstanceOf[V]
+case class Cell[T, A, TM, CM](private val toJs: raw.buildLibTypesMod.Cell[T, A]):
+  lazy val id: CellId                         = CellId(toJs.id)
```

Comment:
Are these opaque types?

Reply:
Yes. Actually, there are new Ids which are not opaque, I will add a commit.